src/server.cpp

```diff
@@ -3112,6 +3121,16 @@ bool Server::setSky(Player *player, const video::SColor &bgcolor,
 	return true;
 }
 
+bool Server::setClouds(Player *player, const float density,
```

Comment:
This line has changed now (Player → RemotePlayer), but the spacing is still the same – which one is problematic? The line seems consistent with the surrounding code…